shared-libs/transitions/src/transitions/utils.js

```diff
@@ -25,8 +25,8 @@ module.exports = {
       errorMessage = messages.getMessage(config, utils.getLocale(doc));
       message = config;
     } else {
-      errorMessage = `messages.generic.${errorKey}`;
-      message = { translation_key: errorMessage };
+      errorMessage = utils.translate(`messages.generic.${errorKey}`, utils.getLocale(doc));
+      message = { translation_key: `messages.generic.${errorKey}` };
```

Comment:
hmm, I think the whole point to add the `translation_key` was so the message was translated later. We shouldn't need to be translating before! 
If your message doesn't end up being translated, we have a different problem. we shouldn't be needing this change.

Reply:
The message sent to the user is translated correctly. The issue is the error message as you correctly noted on the top level comment.